README.md

```diff
@@ -97,3 +98,42 @@ since the commit hash is used as the image tag.
    ```bash
       hubploy deploy <hub-name> hub prod
    ```
+
+## How to use an unreleased helm chart
+
+Sometimes you might need to use the latest version of a helm chart and that isn't yet released.
+The following steps shows how to do so for the DaskHub project, but the pattern
+should be replicable for others.
+
+1. Clone the repo holding the latest version of the helm-chart and checkout the wanted branch:
+
+   *DaskHub example:*
+
+   ```bash
+      git clone https://github.com/dask/helm-chart.git
+      git checkout master
+   ```
+2. Inside the `yaml` file holding the dependencies, add the local path to the helm-chart of the project
+you just checked out and use the version inside the `chart.yaml` or run `chartpress` to generate one if
+that is not available:
+
+   *DaskHub example:*
+
+   ```bash
+      dependencies:
+       - name: daskhub
+         repository: file://<path-to-your-daskhub-checkout>/daskhub
+         version: <version-from-chart.yaml-of-daskhub>
+   ```
+
+3. Run an update any time the dependencies (in `requirements.yaml` or `Chart.yaml`) of the chart you
+just cloned changes.
+
+      *DaskHub example:*
+
+      ```bash
+         # from inside the dask/helm-chart checkout
+         helm dep up daskhub
+      ```
```

Comment:
```suggestion
      ```bash
      # from inside the dask/helm-chart checkout
      helm dep up daskhub
      ```
```